src/components/Difficulty.tsx

```diff
@@ -0,0 +1,14 @@
+function Difficulty() {
+  return (
+    <div className="select-difficulty">
+      <div>Choose difficulty</div>
+      <select>
+        <option value="light">light</option>
+        <option value="medium">medium</option>
+        <option value="hard">hard</option>
+      </select>
+    </div>
+  );
+}
+
+export default Difficulty;
```

Comment:
let's make a generic SelectComponent 'cause now there are duplications in Difficulty, Category, Time, and others, however these components have the same structure: div as a wrapper, title and a select
let's think how we can handle that :)

Reply:
no problem, I will menage with it